Dockerfile

```diff
@@ -0,0 +1,67 @@
+# This is a multi-stage build which requires Docker 17.05 or higher
+FROM python:3.7-alpine as molecule-builder
+
+WORKDIR /usr/src/molecule
+
+ENV PACKAGES="\
+    gcc \
+    git \
+    libffi-dev \
+    make \
+    musl-dev \
+    openssl-dev \
+    "
+RUN \
+    apk update && \
+    apk add --update --no-cache ${PACKAGES}
+
+ADD . .
+RUN \
+    pip wheel \
+    -w dist \
+    .[azure,docker,docs,ec2,gce,lxc,openstack,vagrant,windows]
+
+# ✄---------------------------------------------------------------------
+# This is an actual target container:
+
+FROM python:3.7-alpine
+LABEL maintainer "Ansible <info@ansible.com>"
+
+ENV PACKAGES="\
+    docker \
+    ruby \
+    "
+ENV BUILD_DEPS="\
+    gcc \
+    libc-dev \
+    make \
+    ruby-dev \
+    ruby-rdoc \
+    "
+
+ENV PIP_INSTALL_ARGS="\
+    --only-binary :all: \
+    --no-index \
+    -f /usr/src/molecule/dist \
+    "
+ENV PIP_PACKAGES="\
+    molecule[azure,docker,docs,ec2,gce,lxc,openstack,vagrant,windows] \
+    "
+
+ENV GEM_PACKAGES="\
+    rubocop \
```

Comment:
Also need `inspec`? https://github.com/ansible/molecule/blob/master/molecule/verifier/inspec.py

Reply:
- locally, molecule will only lint the ruby code with rubocop, and won't exec the inspec tests  https://rubocop.readthedocs.io/en/latest/
- inspec is needed only on the target hosts, to execute the tests

Reply:
Thanks!